Gems/Atom/RPI/Code/Source/RPI.Reflect/Model/ModelAsset.cpp

```diff
@@ -134,7 +134,7 @@ namespace AZ
         void ModelAsset::BuildKdTree() const
         {
             AZStd::lock_guard<AZStd::mutex> lock(m_kdTreeLock);
-            if (m_isKdTreeCalculationRunning == false)
+            if ((m_isKdTreeCalculationRunning == false) && !m_kdTree)
```

Comment:
would this be better?
```suggestion
            if (!m_isKdTreeCalculationRunning && !m_kdTree)
```